src/convert-workspace.ts

```diff
@@ -42,12 +43,23 @@ export const GIT_STAGING_BRANCH_NAME = 'polymer-modulizer-staging';
 function writePackageJson(repo: WorkspaceRepo, packageVersion: string) {
   const bowerPackageName = path.basename(repo.dir);
   const bowerJsonPath = path.join(repo.dir, 'bower.json');
-  const bowerJson = readJson(bowerJsonPath);
+  const bowerJson = fse.readJSONSync(bowerJsonPath) as Partial<BowerConfig>;
   const npmPackageName =
       lookupNpmPackageName(bowerJsonPath) || bowerPackageName;
-  const packageJson =
-      generatePackageJson(bowerJson, npmPackageName, packageVersion);
-  writeJson(packageJson, repo.dir, 'package.json');
+
+  const packageJsonPath = path.join(repo.dir, 'package.json');
+  let existingPackageJson: Partial<YarnConfig>|undefined;
+  if (fse.pathExistsSync(packageJsonPath)) {
```

Comment:
This loading code is basically duplicated. Consider putting it in the generate function.

Reply:
I added `util.readJsonIfExists<T>` LMKWYT! Didn't really feel we should to add an FS dependency to this function.